src/applications/dispute-debt/actions/index.js

```diff
@@ -63,6 +63,10 @@ export const fetchDebts = async dispatch => {
       label: getDebtLabel(debt),
       description: getDebtDescription(debt),
       debtType: DEBT_TYPES.DEBT,
+      deductionCode: debt.deductionCode,
+      currentAr: debt.currentAr,
+      originalAr: debt.originalAr,
+      benefitType: debt.benefitType,
     }));
```

Comment:
Not sure if all of this is needed, deductionCode was definitely required. I added the others just in case:
 ``` js
      currentAr: debt.currentAr,
      originalAr: debt.originalAr,
      benefitType: debt.benefitType,
```

Reply:
We may be able to leave this off and make better use of `getDebtLabel` from further up in the same file. That generates the labels we use on debt selection with "X overpayment for Y deduction"